spec/controllers/holds_controller_spec.rb

```diff
@@ -350,14 +354,38 @@
         get :result, params: {}, session: { place_hold_catkey: '1', place_hold_results: results }
 
         placed_hold = assigns(:place_hold_results)[:success].first[:placed_hold]
-        expect(placed_hold.record.first['key']).to eq 'a_hold_key'
+        expect(placed_hold.record['key']).to eq 'a_hold_key'
       end
 
       it 'assigns failed holds results correctly' do
         get :result, params: {}, session: { place_hold_catkey: '1', place_hold_results: results }
 
         failed_hold = assigns(:place_hold_results)[:error].first[:failed_hold]
-        expect(failed_hold.record.first['fields']['barcode']).to eq 'not_holdable_barcode'
+        expect(failed_hold.record['fields']['barcode']).to eq 'not_holdable_barcode'
+      end
+
+      context 'when hold lookup does not return item level info' do
+        let(:hold_info_empty_item_response) { HOLD_LOOKUP_NIL_ITEM_RAW_JSON.to_json }
```

Comment:
Think this is worth it? Follows pattern from 318

```suggestion
      context 'when hold lookup does not return item level info' do
        # HOLD_LOOKUP_NIL_ITEM_RAW_JSON is a mocked SymphonyClient response
        let(:hold_info_empty_item_response) { HOLD_LOOKUP_NIL_ITEM_RAW_JSON.to_json }
```